test/integration/test-pipe.js

```diff
@@ -14,10 +14,26 @@ var remoteFile = 'http://localhost:'+common.staticPort+'/unicycle.jpg';
 // just to deal with ReadStream "autostart"
 // Can't wait for 0.10
 var FIELDS = [
-  {name: 'my_field', value: 'my_value'},
-  {name: 'my_buffer', value: function(){ return new Buffer([1, 2, 3])} },
-  {name: 'my_file', value: function(){ return fs.createReadStream(common.dir.fixture + '/unicycle.jpg')} },
-  {name: 'remote_file', value: function(){ return request(remoteFile)} }
+  {
+    name: 'my_field',
+    type: FormData.DEFAULT_CONTENT_TYPE,
```

Comment:
Great! More verbose where it needs to be and more concise where details not important. 

I think we don't need `type` for this one, it's not a file, so we don't send Content-Type along and we don't check for it. Am I right?

Other than that it's :+1: